nltk/test/unit/test_stem.py

```diff
@@ -15,14 +15,16 @@ def test_arabic(self):
         this unit testing for test the snowball arabic light stemmer
         this stemmer deals with prefixes and suffixes
         """
-        ar_stemmer = SnowballStemmer("arabic")
+        ar_stemmer = SnowballStemmer("arabic", True)
```

Comment:
Please add another test where the `ignore_stopwords=False`.

Reply:
WIP